nservicebus/upgrades/5to6-moving-away-from-ibus.md

```diff
@@ -8,11 +8,11 @@ Starting with Version 6, the `IBus` interface has been deprecated.
 The `IBus` interface was one of the key interfaces when using previous versions of NServiceBus. It provided access to many of the operations like sending messages, subscribing to events, and manipulating headers. The `IBus` interface was available through dependency injection in message handlers, any custom components that were registered in the container and through other mechanisms like the saga base class.
 
 
-## Why was IBus deprecated?
+## Reasons for deprecating IBus
 
-The problem with `IBus` is that the interface contained numerous methods and properties. However, not all of them are valid in certain scenarios. For example, methods like `Reply` and `ForwardCurrentMessageTo` are always available on the `IBus` interface, but it only makes sense to use those in the context of handling an incoming message. Using them in other scenarios will throw an exception.
+The `IBus` interface contained numerous methods and properties. However, not all of them are valid in all scenarios where an instance of `IBus` was exposed and this can lead to confusion. For example, the methods `Reply` and `ForwardCurrentMessageTo` are always available on the `IBus` interface, but it is only relevant to use it in the context of handling an incoming message. Using them in other scenarios will throw an exception.
```

Comment:
>but it is only relevant to use it in the context of handling an incoming message.

The subject are the methods not the bus, I would change to:

>but they are only relevant to use in the context of handling an incoming message.

Switching from **it** to **they** also ties this sentence better into the following one.